src/program.cpp

```diff
@@ -1012,6 +1019,11 @@ cl_build_status cvk_program::do_build_inner_offline(bool build_to_ir,
     std::string clspv_input_file{tmp_folder + "/source"};
     // Save input program to a file
     if (build_from_il) {
+#ifndef ENABLE_SPIRV_IL
+        cvk_error_fn(
+            "Trying to build SPIR-V IL while SPIR-V IL has been disabled");
```

Comment:
I'd rather you use a more explicit message as above. It might not be clear to a user what "SPIR-V IL has been disabled means" and the error message is not as actionable.